services/vaadin-directory/vaadin-directory-rating.service.js

```diff
@@ -0,0 +1,62 @@
+'use strict'
+
+const { starRating } = require('../text-formatters')
+const { floorCount: floorCountColor } = require('../color-formatters')
+const { BaseVaadinDirectoryService } = require('./vaadin-directory-base')
+
+module.exports = class VaadinDirectoryRating extends BaseVaadinDirectoryService {
+  static get category() {
+    return 'rating'
+  }
+
+  static get route() {
+    return {
+      base: 'vaadin-directory',
+      pattern: ':which(star|stars|rating)/:packageName',
+    }
+  }
+
+  static get examples() {
+    return [
+      {
+        title: 'Vaadin Directory',
+        pattern: 'rating/:packageName',
+        namedParams: { packageName: 'vaadinvaadin-grid' },
+        staticPreview: this.render({ which: 'rating', score: 4.75 }),
+        keywords: ['vaadin-directory', 'rating'],
+      },
+      {
+        title: 'Vaadin Directory',
+        pattern: 'stars/:packageName',
+        namedParams: { packageName: 'vaadinvaadin-grid' },
+        staticPreview: this.render({ which: 'star', score: 4.75 }),
+        keywords: ['vaadin-directory', 'star', 'stars'],
```

Comment:
What would you think about a single example with a `:which(rating|stars)/:packageName`? That would make it easier for a user to toggle between the two formats from within the window without having to go back and forth between the editor and listing page. 

Not a blocker, but wanted to suggest it along the lines of the goals proposed in #3174